qiita_ware/test/test_ebi.py

```diff
@@ -530,7 +578,7 @@ def test_generate_demultiplexed_fastq(self):
       <STUDY_TITLE>
         Identification of the Microbiomes for Cannabis Soils
       </STUDY_TITLE>
-      <STUDY_TYPE existing_study_type="Metagenomics"/>
+      <STUDY_TYPE existing_study_type="Metagenomics" />
```

Comment:
You mean the xml library? Why was that not there before?

On (Sep-04-15|11:49), Antonio Gonzalez wrote:
> > @@ -530,7 +578,7 @@ def test_generate_demultiplexed_fastq(self):
> >        <STUDY_TITLE>
> >          Identification of the Microbiomes for Cannabis Soils
> >        </STUDY_TITLE>
> > -      <STUDY_TYPE existing_study_type="Metagenomics"/>
> > +      <STUDY_TYPE existing_study_type="Metagenomics" />
> 
> Yes, from the tool doing the format. Note that doesn't add/remove anything.  
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1464/files#r38782763